lib/protocol/SqlString.js

```diff
@@ -66,16 +66,16 @@ SqlString.arrayToList = function(array, timeZone) {
 
 SqlString.format = function(sql, values, stringifyObjects, timeZone) {
   values = values == null ? [] : [].concat(values);
```

Comment:
Ouch, I missed that it can be an object too. Then there's nothing to gain here. My bad.